IORequestGenerator/IORequestGenerator.cpp

```diff
@@ -1013,7 +1013,7 @@ DWORD WINAPI threadFunc(LPVOID cookie)
     // adjust thread token if large pages are needed
     for (auto pTarget = p->vTargets.begin(); pTarget != p->vTargets.end(); pTarget++)
     {
-        if (pTarget->GetUseLargePages())
+        if (pTarget->GetUseLargePages() | pTarget->GetIOBufferAlignmentSpecified())
```

Comment:
Shouldn't this be defined as || and not |?

Reply:
Both would work but yes, I'd agree as a matter of practice that using the Boolean test is preferable.